Documentation/source/Getting started.md

```diff
@@ -45,3 +46,115 @@ let documentService: AccountingDocumentService = giniPayApiLib.documentService()
+
+> ⚠️  **Warning**
+> - If you are using the Gini Accounting API, then composite documents are not supported. You need to use `documentService.fetchDocument()` and then `documentService.extractions()` without creating a composite document.
+
+Before retrieving extractions you need to create a composite document from your partial documents
+
+Gini needs to process the composite document first before you can fetch the extractions. Effectively this means that you won’t get any extractions before the composite document is fully processed. The processing time may vary, usually it is in the range of a couple of seconds, but blurred or slightly rotated images are known to drasticly increase the processing time.
+
+The `DocumentService` provides `extractions()` method which can be used to fetch the extractions after the processing of the document is completed. The following example shows how to achieve this in detail.
+
+```swift
+documentService
+    .createDocument(fileName: "composite-myFirstDocument.jpg",
+                    docType: nil,
+                    type: .composite(CompositeDocumentInfo(partialDocuments: documents)),
+                    metadata: metadata) { [weak self] result in
+    guard let self = self else { return }
+    switch result {
+    case let .success(createdDocument):
+        print("🔎 Starting analysis for composite document with id \(createdDocument.id)")
+        self.documentService.extractions(for: createdDocument,
+                           cancellationToken: CancellationToken()) { result in
+            DispatchQueue.main.async {
+                switch result {
+                case let .success(extractionResult):
+                    completion(.success(extractionResult.extractions))
+                    print("The specific extractions: \(extractionResult.extractions)")
+                    print("The line item compound extractions : \(extractionResult.lineItems)")
+                case let .failure(error):
+                    completion(.failure(.apiError(error)))
+                }
+            }
+        }
+    case let .failure(error):
+        print("❌ Composite document creation failed: \(error)")
+        completion(.failure(error))
+    }
+}
+```
+
+### Sending feedback
+
+Depending on your use case your app probably presents the extractions to the user and gives them the opportunity to correct them. Yes, there could be errors. We do our best to prevent them - but it’s more unlikely to happen if your app sends us feedback for the extractions we have delivered. Your app should send feedback only for the extractions the user has seen and accepted. Feedback should be sent for corrected extractions and for correct extractions. The code example below shows how to correct extractions and send feedback.
```

Comment:
Could you break this up a bit. It's too much info in one paragraph. I also changed the wording to sound more professional:
```suggestion
Depending on your use case your app probably presents the extractions to the user and gives them the opportunity to correct them. By sending us feedback for the extractions we are able to continuously improve the extraction quality.

Your app should send feedback only for the extractions the user has seen and accepted. Feedback should be sent for corrected extractions and for correct extractions. The code example below shows how to correct extractions and send feedback.

```